test/e2e/specs/authentication/authentication__totp.ts

```diff
@@ -2,21 +2,54 @@
  * @group authentication
  */
 
-import { DataHelper, TestAccount } from '@automattic/calypso-e2e';
+import { DataHelper, LoginPage, SecretsManager, TOTPClient } from '@automattic/calypso-e2e';
 import { Page, Browser } from 'playwright';
 
 declare const browser: Browser;
 
 describe( DataHelper.createSuiteTitle( 'Authentication: TOTP' ), function () {
+	const credentials = SecretsManager.secrets.testAccounts.totpUser;
 	let page: Page;
-	let testAccount: TestAccount;
+	let loginPage: LoginPage;
 
 	beforeAll( async () => {
 		page = await browser.newPage();
 	} );
 
-	it( 'Authenticate as user with TOTP 2FA code', async function () {
-		testAccount = new TestAccount( 'totpUser' );
-		await testAccount.authenticate( page );
+	// This spec intentionally manually calls individual
+	// methods from LoginPage to separate out the steps that
+	// are bundled together in the TestAccount class.
+	it( 'Navigate to Login page', async function () {
+		await page.goto(
+			DataHelper.getCalypsoURL(
+				'log-in?site=e2eflowtestingtotp.wordpress.com&redirect_to=%2Fsettings%2Fgeneral'
```

Comment:
Potentially want to test a full url as well as a relative one in the redirect_to param. Fine to push back and suggest that's better to be covered in a unit tests though :smile:

Reply:
It looks like there's already a little bit of [unit test coverage for redirect_to here](https://github.com/Automattic/wp-calypso/blob/12bd8f9769e0f47f35e3126d7bbb46de372aae2b/client/login/test/ssr.js), so if we could flesh that out a little bit I would prefer that over adding to e2e.

Reply:
I'd say we should still have a redirect as the unit tests did not catch the recent subject of the post-mortem.

Unrelated, I realized the test user's site is shown in the siteSlug portion in the snippet here. I have to remove that.

Reply:
I've fixed the siteSlug for the test user showing up in the full URL.

Thinking about this a bit further, I think tests for the `/login` endpoint should have coverage at every level if possible. This could either be a 20% time project, or a full project led by KitKat.